MovieSharing/VideoModel.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Video.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 28.09.21.
+//
+
+import Foundation
+
+struct VideosResponse:Decodable {
+    
+    var items: [Video]
+    
+}
+
+struct Video:Decodable {
+    
+    var id: String
+    var snippet:Snippet
+    
+}
+
+
+struct Snippet:Decodable {
+    
+    var publishedAt: String
+    var title:String
+        var description: String
+        var Thumbnails: Thumbnail
+    
+}
+
+struct Thumbnail:Decodable {
+    
+    var default_image: ImageObject
```

Comment:
for vars we use camel case, so would be good to rename these to defaultImage etc.